src/remote_engine_client/src/client.rs

```diff
@@ -140,33 +144,35 @@ impl Client {
         // Write to remote.
         let table_ident = request.table.clone();
         let endpoint = route_context.endpoint.clone();
-        let request_pb = request.convert_into_pb().box_err().context(Convert {
-            msg: "Failed to convert WriteRequest to pb",
+        let request_fb = request.convert_into_fb().box_err().context(Convert {
+            msg: "Failed to convert WriteRequest to fb",
         })?;
-        let mut rpc_client = RemoteEngineServiceClient::<Channel>::new(route_context.channel);
+
+        let mut rpc_client = RemoteEngineFbServiceClient::<Channel>::new(route_context.channel);
```

Comment:
Does this mean remote server will only have request with fb payload?

Reply:
Adding a REQUEST_TYPE environment variable to support both protobuf and flatbuffer protocol.